manifests/bucketeer/values.yaml

```diff
@@ -2,3 +2,9 @@ global:
   image:
     tag: v0.4.5 # x-release-please-version
     imagePullSecrets:
+
+migration:
+  image:
+    repository: ghcr.io/bucketeer-io/bucketeer-migration
+    tag: v0.4.5 # x-release-please-version
```

Comment:
Set the default tag and the comment so the Release Please can update the tag when releasing.